### ISSUE TITLE
Feedback concept Ubi

### ISSUE THREAD

**Issue by shrls** [Murderlon/ubicomp#1]
Mooi idee! Heb je alle componenten al in huis? Zo niet, dan is er een goedkoop alternatief voor de dure pressure sensor: http://www.instructables.com/id/How-To-Use-Touch-Sensors-With-Arduino/. Ook kan het zijn dat ik nog een elektret heb, moet ik nakijken. 

Ik zou aan de ux kant nog willen weten of het apparaat mobiel is - kun je er net zoals een geigerteller mee rondlopen om de perfecte plek te vinden? In dat geval zou er affordance moeten aangebracht worden om dat mogelijk te maken. Of is 'ie bedoeld om op een tafel te staan? Nu nog niet helemaal duidelijk. 

De relevance-stap in de feedback loop zit er goed in, maar de consequence minder -- wat kan een werknemer doen als de condities slecht zijn? Kun je 'm in de goede richting wijzen? Dat kan je project nog wat beter maken. 

Succes!

**Comment by Murderlon** [comment]
@shrls 
> Ik zou aan de ux kant nog willen weten of het apparaat mobiel is - kun je er net zoals een geigerteller mee rondlopen om de perfecte plek te vinden?

Yes! Ik dacht aan een soort portable charger die in het doosje zit.

> wat kan een werknemer doen als de condities slecht zijn? Kun je 'm in de goede richting wijzen?

Dat klinkt al lastiger, maar ik ga ik nog even over na denken.
